app/main.py

```diff
@@ -16,4 +16,18 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+
+
+def sort_variables(*args) -> dict:
+    for value in args:
+        if isinstance(value, (int, str, bool, float, tuple)):
+            sorted_variables["immutable"].append(value)
+        else:
+            sorted_variables["mutable"].append(value)
+    return sorted_variables
+
+
+sort_variables(lucky_number, pi, one_is_a_prime_number, name,
+               my_favourite_films, profile_info, marks,
+               collection_of_coins)
```

Comment:
This implementation works, but it's not the same as stated in the task, you have just to create a dictionary with mutable and immutable lists, without any other logic

Reply:
completed the task according to the instructions